### ISSUE TITLE
Check for links that open in a new tab or a new window

### ISSUE THREAD

**Issue by jackdomleo7** [jackdomleo7/Checka11y.css#10]
While the act of opening in a new window or tab isn't an accessibility concern itself, according to W3, it's essential to provide advanced warning to a user when a link opens in a new window or tab.

The A11y Project link:https://www.a11yproject.com/checklist/#identify-links-that-open-in-a-new-tab-or-window
W3 link: https://www.w3.org/TR/WCAG20-TECHS/G201.html

**Solution**

Provide only a warning message when the following is detected:
```css
a[target="_blank"] { ... }
```

**Comment by devindford** [comment]
Working on this!
